src/ochanticipy/datasources/usgs/ndvi_base.py

```diff
@@ -102,15 +98,19 @@ def __init__(
         self._data_variable_suffix = data_variable_suffix
 
         # set dates for data download and processing
-        self._start_date = get_dekadal_date(
-            input_date=start_date, default_date=_EARLIEST_DATE
+        self._start_date = get_kalendar_date(
```

Comment:
Why didn't you keep `get_dekadal_date` here?

Reply:
hahaha, because I am a dingus. I didn't have `get_dekadal_date()` at first.